program-runtime/src/invoke_context.rs

```diff
@@ -154,12 +164,14 @@ impl<'a> InvokeContext<'a> {
             blockhash,
             lamports_per_signature,
             syscall_context: Vec::new(),
+            enable_instruction_tracing: debugging_features,
         }
     }
 
-    pub fn new_mock(
+    pub fn new_mock_with_debugging(
```

Comment:
I don't see a reason why a mockup would disable debugging. So just hard code it to `true` here.

Reply:
@Lichtso thanks for your note. As I've researched, the only place where debugging is enabled is rbpf-cli. All other places create loader without debugging and instruction tracing, so I tried not to conflict with this behavior.

Reply:
yes you are right, it would increase the time CI takes and that is not good.
Maybe just have it be `false` by default and have a `InvokeContext::activate_debugging()` method which sets it to `true`?

Reply:
Okay, I will update this like you're suggesting.

Reply:
BTW, it is not only CI problem, it can also affect performance of validators.

Reply:
@Lichtso done. :)